components/pedigree/resources/src/main/resources/pedigree/view/person.js

```diff
@@ -1579,6 +1579,8 @@ define([
                     //          {id: 'ENSG00000160688', gene: 'FLAD1', status: 'solved'},
                     //          {id: 'custom', 'gene': 'custom', status: 'candidate'} ]
                     this._setGenes(info.genes);
+                } else {
+                    this._setGenes([]);
```

Comment:
@veronikaslc we shouldn't have to specify the getters/setters in the map, they should all be standard like in Java: `get<CapitalizedFieldName>`. Also, since some setters trigger a possibly costly ui update, the check whether the new value is different from the old one makes more sense in the setter imo, so that all other code can just call the setter without worrying. Just my 2cents.